src/clarity-angular/datagrid/providers/filters.spec.ts

```diff
@@ -74,21 +74,21 @@ export default function(): void {
 
         it("offers a way to unregister a filter", function() {
             let filter = new EvenFilter();
-            let unregister = this.filtersInstance.add(filter);
+            let registeredFilter = this.filtersInstance.add(filter);
             let nbChanges = 0;
             this.filtersInstance.change.subscribe(() => nbChanges++);
             filter.toggle();
             expect(this.filtersInstance.getActiveFilters()).toEqual([filter]);
             expect(nbChanges).toBe(1);
-            unregister();
+            registeredFilter.unregister();
             expect(this.filtersInstance.getActiveFilters()).toEqual([]);
             filter.toggle();
-            expect(nbChanges).toBe(1);
+            expect(nbChanges).toBe(2); // first is toggle(). second is unregister()
```

Comment:
It took me a little while to understand what was happening there. The unregister triggers a change _because the filter was active_. So here, to make this clear, you just need to add another expect:
```typescript
registeredFilter.unregister();
expect(this.filtersInstance.getActiveFilters()).toEqual([]);
expect(nbChanges).toBe(2);
filter.toggle();
expect(nbChanges).toBe(2);
```
It makes it clear that the toggle isn't the cause of the extra change.

That said, I would really like to have two unit tests for this "unregister triggers a change" behavior:
1) Unregister an inactive filter, no change expected.
2) Unregister an active filter, change expected.